lib/validation.js

```diff
@@ -13,6 +13,8 @@ const {
   FST_ERR_SCH_RESPONSE_SCHEMA_NOT_NESTED_2XX
 } = require('./errors')
 
+const warning = require('./warnings')
```

Comment:
need a test for it - what if I have 2 routes with undefined schema? does the warning show up once?

In this case I think it should be emitted for every route

Reply:
Hello @Eomm, it is emitted only once. As far as I see, [process-warning](https://github.com/fastify/process-warning) guarantees that the same warning is emitted only once.

We can't have a warning emitted for every route if we use the same warning code for all the routes. Perhaps, we could emit a warning with a dynamic code every time, but it would imply:

1. Creating the warning directly on `validations.js`.
2. The warning code would be dynamic, so we need to find a way to avoid collisions to emit still the warning for every route with "_undefined_ schema".

Please, note it is the first time I'm working on the Fastify codebase, so please, correct me if I'm wrong or if there is another simpler way than my previous thought.

Reply:
https://github.com/fastify/process-warning/pull/78 👍🏼 

You will need to add a boolean and bump the module whenever we release it please 🙏🏼

Reply:
Any news for the new release of the `process-warning`? Thank you

Reply:
Done: https://github.com/fastify/process-warning/releases/tag/v2.2.0

Reply:
Perfect, I've just updated the code using the new version and the new `unlimited` flag enabled. Note that the validation warning is emitted twice for GET routes, one for the method GET and another for the method HEAD.

> [FSTDEP015] FastifyDeprecation: The body schema for GET: /undefinedBody/:id is missed. This may indicate the schema is not well specified.
[FSTDEP015] FastifyDeprecation: The body schema for HEAD: /undefinedBody/:id is missed. This may indicate the schema is not well specified.